core-bundle/contao/themes/flexible/styles/layout/list-view.pcss

```diff
@@ -279,22 +279,31 @@
     display: flex;
     justify-content: flex-end;
     align-items: center;
-    gap: 3px;
```

Comment:
This one is actually needed for the gap in the theme overview:
 
 Current:
 
<img width="198" height="85" alt="image" src="https://github.com/user-attachments/assets/5541a715-2bc4-45d2-9a48-0c5ddb47ca97" />

This PR:
<img width="233" height="103" alt="image" src="https://github.com/user-attachments/assets/c354bc8f-bc39-4301-bfa9-51d0cd1671a5" />

It also affects tl_log, undo, etc (Basically everything that does not have `.tl_right`